tests/unit/helpers.js

```diff
@@ -63,6 +63,7 @@ export function getFakeAddonManagerWrapper({
 }
 
 export function unexpectedSuccess() {
+  // TODO: fix this so that the failure message is better.
```

Comment:
This is coming in the next patch. The problem is that `expect(false).toBe(true)` produces a terrible error message that doesn't help you figure out what went wrong.